Automattic/vip-go-mu-plugins
# Move VIP ES setup code to the new class

## Description

Now that we have the `Automattic\VIP\Elasticsearch\Elasticsearch` class, move the existing constant/filter setup code to it.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Pull down PR
1. Use the site without setting `USE_VIP_ELASTICSEARCH` constant - class shouldn't load, index names aren't filtered, etc
1. Set `USE_VIP_ELASTICSEARCH` to `true` and ensure the class loads and sets the constants / filters
